Rename `racket' to `minimal-racket'

leafac opened the pull request:
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

See https://github.com/caskroom/homebrew-cask/pull/23508#issuecomment-238556262.

MikeMcQuaid commented:
Nice work but can you check out these docs: https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Rename-A-Formula.md

Thanks!

leafac commented:
@MikeMcQuaid: Thanks for the quick feedback. I read the documentation you linked; added an entry to `formula_renames.json` and fixed the commit message.

Also, I recommend adding a link to that on the Pull Request template. Maybe under a section “Rename a Formula”, like the [Homebrew Cask Pull Request template](https://github.com/caskroom/homebrew-cask/blob/8b53b4e9473b9dde6d30b22dc86d5368ead06195/.github/PULL_REQUEST_TEMPLATE.md) is organized in sections.

MikeMcQuaid commented on formula_renames.json:
```diff
@@ -24,5 +24,6 @@
   "screenbrightness": "brightness",
-  "wires": "geckodriver"
+  "wires": "geckodriver",
+  "racket": "minimal-racket"
```
Sorry for pedantry but can you put this in alphabetical order? Thanks!

jawshooah replied:
The rest of the file doesn't seem to be sorted 😞

leafac replied:
@MikeMcQuaid, @jawshooah: Thank you for the feedback.

@jawshooah correctly pointed out that the file currently is not in order. So you want me to sort the whole thing? Do I sort in terms of the old name (the key) or the new one (the value)? Shouldn't I do it in a separate Pull Request?

MikeMcQuaid replied:
Sorted for next time in 2e4af01.

MikeMcQuaid commented:
May also want to make an alias for `racket`

leafac commented:
@MikeMcQuaid: I think having an alias for `racket` defeats the purpose of https://github.com/caskroom/homebrew-cask/pull/23508.

MikeMcQuaid commented:
@leafac Fair enough 👍.

MikeMcQuaid commented:
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

leafac commented:
@MikeMcQuaid: That _you_ for helping maintain Homebrew 👍
